File: cjisAPI/Juror.cs
using System;
using System.Data.SqlClient;
using System.Data;

namespace cjisAPI
{
	public class Juror {
		/***properties***/
		public int JurorID { get; set; }
		public string FirstName { get; set; }
		public string MiddleInitial { get; set; }
		public string LastName { get; set; }
		public string Address1 { get; set; }
		public string Address2 { get; set; }
		public DateTime DateOfBirth { get; set; }
		public string T { get; set; }
		public string G { get; set; }
		public int CityID { get; set; }
		public int StateID { get; set; }
		public int ZIP { get; set; }
		public string SocialSecurityNumber { get; set; }
		public string HomePhone { get; set; }
		public string WorkPhone { get; set; }
		public string Race { get; set; }
		public string Sex { get; set; }
		public bool Hispanic { get; set; }
		public string Source { get; set; }
		public int VoterRegistrationID { get; set; }
		public int DriverLicenseNumber { get; set; }
		public int JuryBoxID { get; set; }
		public int NonCitizen { get; set; }
		public string Comment { get; set; }
		public int NonCitizenID { get; set; }
		public bool QuestionnaireCompleted { get; set; }

		public Juror(int jurorId) {
			JurorID = jurorId;

			GetData();
		}

		private void GetData() {
			DataCommand command = new DataCommand("spGetJuror", CommandType.StoredProcedure);
			command.AddParameter("@jurorId", JurorID);

			SqlDataReader reader = command.ExecuteReader();

			if (reader.Read()) {
				JurorID = reader.GetInt32(0);
				FirstName = reader.GetString(1);
				//MiddleInitial = reader.GetString(2);
				LastName = reader.GetString(3);
				//Address1 = reader.GetString(4);
				//Address2 = reader.GetString(5);
				//DateOfBirth = reader.GetDateTime(6);
				//T = reader.GetString(7);
				//G = reader.GetString(8);
				//CityID = reader.GetInt32(9);
				//StateID = reader.GetInt32(10);
				//ZIP = reader.GetInt32(11);
				//SocialSecurityNumber = reader.GetString(12);
				//HomePhone = reader.GetString(13);
				//WorkPhone = reader.GetString(14);
				//Race = reader.GetString(15);
				//Sex = reader.GetString(16);
				//Hispanic = reader.GetBoolean(17);
				//Source = reader.GetString(18);
				//VoterRegistrationID = reader.GetInt32(19);
				//DriverLicenseNumber = reader.GetInt32(20);
				//JuryBoxID = reader.GetInt32(21);
				//NonCitizen = reader.GetInt32(22);
				//Comment = reader.GetString(23);
				//NonCitizenID = reader.GetInt32(24);
				//QuestionnaireCompleted = reader.GetBoolean(25);
			}

			command.Close();
		}

		public static Juror GetJuror(int jurorId) {
			return new Juror(jurorId);
		}
	}
}

File: test-app/src/Models/Juror.js
import * as PropTypes from 'prop-types';
import Questionairre from './Quetionairre';

export default class Juror {
    constructor(juror ={}) {
        const {
        jurorID = null,
        firstName = null,
        middleInitial = null,
        lastName = null,
        address1 = null,
        address2 = null,
        dateOfBirth = null,
        t = null,
        g = null,
        cityID = null,
        stateID = null,
        zip = null,
        socialSecurityNumber = null,
        homePhone = null,
        workPhone = null,
        race = null,
        sex = null,
        hispanic = null,
        source = null,
        voterRegistrationID = null,
        driversLicenseNumber = null,
        juryBoxID = null,
        noncitizen = null,
        comment = null,
        questionairreCompleted = null    
        
        } = juror        
        this.jurorID = jurorID;
        this.firstName = firstName;
        this.middleInitial = middleInitial;
        this.lastName = lastName;
        this.address1 = address1;
        this.address2 = address2;
        this.dateOfBirth = dateOfBirth;
        this.t = t;
        this.g = g;
        this.cityID = cityID;
        this.stateID = stateID;
        this.zip = zip;
        this.socialSecurityNumber = socialSecurityNumber;
        this.homePhone = homePhone;
        this.workPhone = workPhone;
        this.race = race;
        this.sex = sex;
        this.hispanic = hispanic;
        this.source = source;
        this.voterRegistrationID = voterRegistrationID;
        this.driversLicenseNumber = driversLicenseNumber;
        this.juryBoxID = juryBoxID;
        this.noncitizen = noncitizen;
        this.comment = comment;
        this.questionairreCompleted = questionairreCompleted
    }
}

Juror.propTypes = {
    jurorID: PropTypes.number,
    firstName: PropTypes.string,
    middleInitial: PropTypes.string,
    lastName: PropTypes.string,
    address1: PropTypes.string,
    address2: PropTypes.string,
    dateOfBirth: PropTypes.instanceOf(Date),
    t: PropTypes.string,
    g: PropTypes.string,
    cityID: PropTypes.number,
    stateID: PropTypes.number,
    zip: PropTypes.number,
    socialSecurityNumber: PropTypes.string,
    homePhone: PropTypes.string,
    workPhone: PropTypes.string,
    race: PropTypes.string,
    sex: PropTypes.string,
    hispanic: PropTypes.bool,
    source: PropTypes.string,
    voterRegistrationID: PropTypes.number,
    driversLicenseNumber: PropTypes.number,
    juryBoxID: PropTypes.number,
    noncitizen: PropTypes.bool,
    comment: PropTypes.string,
    questionairreCompleted: PropTypes.bool
}